src/hpc-cache/azext_hpc_cache/_validators.py

```diff
@@ -4,13 +4,7 @@
 # --------------------------------------------------------------------------------------------
 import argparse
 from knack.util import CLIError
-
-# pylint: disable=unused-argument
-
-
-def transfer_cache_name(cmd, namespace):
-    namespace.cache_name = namespace.name
-    del namespace.name
+from msrestazure.tools import is_valid_resource_id

```

Comment:
Use "from azure.mgmt.core.tools import is_valid_resource_id" and move it to "validate_storage_account_name_or_id"